src/main/java/net/mcreator/element/types/Armor.java

```diff
@@ -85,6 +85,11 @@
 	public String bootsModelPartR;
 	public String bootsModelTexture;
 
+	public boolean helmetImmuneToFire;
+	public boolean chestplateImmuneToFire;

```

Comment:
Consider renaming chestplate to body to match other parameters

Reply:
Done!